src/OrchardCore.Modules/OrchardCore.Users/Drivers/UserDisplayDriver.cs

```diff
@@ -121,46 +128,8 @@ public override async Task<IDisplayResult> UpdateAsync(User user, UpdateEditorCo
                 await Handlers.InvokeAsync((handler, context) => handler.EnabledAsync(userContext), userContext, _logger);
             }
 
-            if (string.IsNullOrWhiteSpace(model.UserName))
-            {
-                context.Updater.ModelState.AddModelError("UserName", S["A user name is required."]);
-            }
-
-            if (string.IsNullOrWhiteSpace(model.Email))
-            {
-                context.Updater.ModelState.AddModelError("Email", S["An email is required."]);
-            }
-
-            var userWithSameName = await _userManager.FindByNameAsync(model.UserName);
-            if (userWithSameName != null)
-            {
-                var userWithSameNameId = await _userManager.GetUserIdAsync(userWithSameName);
-                if (userWithSameNameId != model.Id)
-                {
-                    context.Updater.ModelState.AddModelError(string.Empty, S["The user name is already used."]);
-                }
-            }
-
-            if (model.UserName != user.UserName && user.UserName == httpContext.User.Identity.Name)
-            {
-                context.Updater.ModelState.AddModelError(string.Empty, S["Cannot modify user name of the currently logged in user."]);
```

Comment:
@Skrypt I need to understand why you added this, as I have removed it, and forced a new cookie when editing the current user... so allowing the user to change his name.

Reply:
AFAIK this has been added when the user edit his info (username & role), so it restricts the editing to specific information